cli/buffcli/src/bin/buff/commands/publish.rs

```diff
@@ -0,0 +1,38 @@
+use std::path::{Path, PathBuf};
+use std::sync::Arc;
+use grpcio::{ChannelBuilder, EnvBuilder};
+use bufflib::protobuffers::buff_grpc::{RegistryServiceClient, RegistryService};
+use bufflib::protobuffers::buff::{PublishRequest, PublishResponse, Package};
+use bufflib::package_metadata::PackageMetadata;
+use bufflib::artifact;
+use bufflib::buff_cli_config::BuffCliConfig;
+#[path = "../settings.rs"]
+mod settings;
+use settings::Settings;
+
+pub fn execute() {
+  let config = BuffCliConfig::new();
+  let settings = Settings::new().unwrap();
+  let registry_url = &settings.buff_grpc_server.url;
+  let env = Arc::new(EnvBuilder::new().build());
+  let channel = ChannelBuilder::new(env).connect(registry_url);
```

Comment:
i think that maybe something like a Registry is a good lib to have to hide all this stuff..